modules/mod_ginger_edit/templates/_admin_tinymce_overrides_js.tpl

```diff
@@ -1,7 +1,19 @@
 /*
 Custom settings to override tiny-init.js.
 */
-tinyInit.language="{{ m.translation.language|default:"en" }}";
-tinyInit.menubar="";
-tinyInit.toolbar="styleselect | bold italic | bullist numlist | removeformat | zlink zmedia | link unlink | code";
-tinyInit.extended_valid_elements="iframe[src|style|width|height|scrolling|marginwidth|marginheight|frameborder]";
+
+if (typeof tinymce_overrides_language === 'undefined' || !tinymce_overrides_language) {
+    tinyInit.language="{{ m.translation.language|default:"en" }}";
+}
```

Comment:
That seperate parameter is needed because most tinyInit options will always return true because they have already been set in this javascript file which is include just before the override templates.

https://github.com/zotonic/zotonic/blob/master/modules/mod_editor_tinymce/lib/js/tinymce-4.5.5/tiny-init.js